docs/reference/access-control-lists.md

```diff
@@ -1,37 +1,52 @@
-### The new ACL model
-As mentioned in our [Security Update](https://docs.lakefs.io/posts/security_update.html#whats-changing) the coming lakeFS version introduces a simplified authorization mechanism based on ACLs.
+---
+layout: default
+title: Access Control Lists (ACL)
+parent: Reference
+description: Access Control Lists (ACL)
+nav_order: 100
+has_children: false
+---
 
-### Permissions
-The new ACL model defines permissions at the group level, 
-we'll review the new permissions compared to their previous representation:
+## Access Control Lists (ACL) in lakeFS
+
+ACLs were introduced in their current form in v0.97 of lakeFS as part of [changes to the security model](/posts/security_update.html#whats-changing) in lakeFS. 
```

Comment:
Not sure of the version, I've not released it yet.

Reply:
I'm confused why the docs are live if the change isn't released yet. Should we remove the doc until it is, or reword the doc to speak in the future tense?